pillar/cove_oc4ids.sls

```diff
@@ -1,3 +1,11 @@
+network:
+  host_id: ocp17
+  ipv4: 176.58.112.127
+  ipv6: "2a01:7e00:e000:04c1::"
```

Comment:
Just noticing I might have mis-documented our IPv6 /64 block. In a couple places in the docs I refer to it as `2a01:7e00:e000:02cc::/64`, but maybe it should only be `2a01:7e00:e000:....` ?

Reply:
This is a bigger issue see discussion on #409